test/addons/buffer-free-callback/test.js

```diff
@@ -4,7 +4,20 @@
 require('../../common');
 var assert = require('assert');
 var binding = require('./build/Release/binding');
-var buf = binding.alloc();
-var slice = buf.slice(32);
-buf = null;
-binding.check(slice);
+
+function check(size) {
+  var buf = binding.alloc(size);
+  var slice = buf.slice(size >>> 1);
+
+  buf = null;
+  binding.check(slice);
+  slice = null;
+  gc();
+  gc();
+  gc();
```

Comment:
It's unclear to me why calling `gc()` three times is required here. @indutny Sorry to pull this up from more than 3 years ago, but do you remember why?